conf/eslint-recommended.js

```diff
@@ -127,6 +127,7 @@ module.exports = {
         "no-invalid-this": "off",
         "no-irregular-whitespace": "error",
         "no-iterator": "off",
+        "no-java-constructor": "error",
```

Comment:
We can't enable rules in `eslint:recommended` until the next major release, so please set this to `"off"` for now.